yarn-project/aztec/src/cli/cli.ts

```diff
@@ -44,7 +44,8 @@ export function injectAztecCommands(program: Command, userLog: LogFn, debugLogge
 
     if (options.sandbox) {
       const sandboxOptions = extractNamespacedOptions(options, 'sandbox');
-      userLog(`${splash}\n${github}\n\n`);
+      const packageVersionString = `aztec-packages version: ${process.env.npm_package_version}`;
+      userLog(`${splash}\n${github}\n${packageVersionString}\n\n`);
```

Comment:
@alexghr at some point we used to print the version but clearly we regressed. Do you know why? or is this ok?

Reply:
I think the version got lost in a refactor. 

I'm not sure reading `$npm_package_version` is the right approach for this because it won't be set if running the script directly (not via npm/yarn).